File: app/src/main/java/com/angogasapps/myfamily/ui/screens/main/holders/VideoNewsViewHolder.java
package com.angogasapps.myfamily.ui.screens.main.holders;

import android.content.Context;
import android.view.View;

import androidx.annotation.NonNull;

import com.angogasapps.myfamily.databinding.VideoNewsViewHolderBinding;
import com.angogasapps.myfamily.models.Family;
import com.angogasapps.myfamily.models.NewsObject;
import com.bumptech.glide.Glide;

public class VideoNewsViewHolder extends BaseNewsViewHolder{
    private VideoNewsViewHolderBinding binding;

    public VideoNewsViewHolder(Context context, @NonNull View itemView) {
        super(context, itemView);
        binding = VideoNewsViewHolderBinding.bind(itemView);
    }

    @Override
    public void update(NewsObject newsObject) {
        binding.textName.setText(Family.getInstance().getNameByPhone(newsObject.getFromPhone()));
        binding.userPhoto.setImageBitmap(Family.getInstance().getMemberImageByPhone(newsObject.getFromPhone()));
//        Glide
//                .with(context)
//                .load(newsObject.getValue())
//                .into(binding.video);
    }
}

File: app/src/main/java/com/angogasapps/myfamily/ui/screens/main/holders/VideoNewsViewHolder.kt
package com.angogasapps.myfamily.ui.screens.main.holders

import android.content.Context
import android.view.View
import com.angogasapps.myfamily.databinding.VideoNewsViewHolderBinding
import com.angogasapps.myfamily.ui.screens.main.holders.BaseNewsViewHolder
import com.angogasapps.myfamily.models.events.NewsObject
import com.angogasapps.myfamily.models.Family

class VideoNewsViewHolder(context: Context, itemView: View) :
    BaseNewsViewHolder(context, itemView) {
    private val binding: VideoNewsViewHolderBinding = VideoNewsViewHolderBinding.bind(itemView)
    override fun update(newsObject: NewsObject) {
        binding.textName.text = Family.getNameByPhone(newsObject.fromPhone)

        //        binding.userPhoto.setImageBitmap(Family.getInstance().getMemberImageByPhone(newsObject.getFromPhone()));
        //        Glide
        //                .with(context)
        //                .load(newsObject.getValue())
        //                .into(binding.video);
    }
}